appeals/api/src/database/schema.prisma

```diff
@@ -696,6 +697,37 @@ model RepresentationAttachment {
   @@id([documentGuid, version])
 }
 
+/// RepresentationRejectionReason model
+/// Contains all possible reasons to reject a comment.
+model RepresentationRejectionReason {
```

Comment:
This may need a `type` column to match the `representationType` - very likely we'll want to have separate reasons for comments, statements, final comments, etc